tox.ini

```diff
@@ -83,7 +83,8 @@ commands =
 [testenv:loader-idyntree]
 deps =
     pytest
-    idyntree >= 5.0.1
+    idyntree >= 8.0.0
+    pin >=2.6.10
```

Comment:
Here is another reason to move `get_package_dirs` to a separate file :sweat_smile:

Reply:
Fixed in https://github.com/robot-descriptions/robot_descriptions.py/pull/24/commits/96f3679c62c393abe3da5c9db952b57329851148